components/Accordion/style.scss

```diff
@@ -0,0 +1,43 @@
+.Accordion {
+  background: rgba(var(--color-bg-page-rgb, white), 1);
+  border: 1px solid var(--color-border-default, rgba(black, 0.1));
+  border-radius: 5px;
+
+  .title {
+    align-items: center;
+    background: white;
+    border: 0;
+    border-radius: 5px;
+    color: var(--color-text-default, #384248);
+    cursor: pointer;
+    display: flex;
+    font-size: 14px;
+    justify-content: space-between;
+    padding: 10px;
+    position: relative;
+    text-align: left;
+    width: 100%;
+  }
+
+  .icon {
```

Comment:
@trishaprile—per [our convo on #948](https://github.com/readmeio/markdown/pull/948#discussion_r1710381622) can we also update these classes to follow our BEM selector conventions?